packages/react-effect/package.json

```diff
@@ -30,9 +30,6 @@
   "engines": {
     "node": ">=12.14.0"
   },
-  "devDependencies": {
```

Comment:
Dev dependency not neeeded because `@types/react-dom` is already in the root package.json's dev dependencies